Assets/Feature01/Scripts/Controller/MovementController.cs

```diff
@@ -0,0 +1,50 @@
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class MovementController : MonoBehaviour
+{
+
+
+   
+   
+    Vector3 playerMoveInput;
+
+    public void PlayerMove(Vector2 movementInput, float speed,Rigidbody _rigidbody)
+    {
```

Comment:
Are you sure you wish to create movement based on rigid body? 
Open ended question. It can be something really fun.. Or something very complicated to manage.